doc/sphinx/addendum/ring.rst

```diff
@@ -1,736 +1,737 @@
-     becomes $N_1/D_1 = N_2/D_2$. A normalization step (the same as the
-     one for rings) is then applied to $N_1$, $D_1$, $N_2$ and
-     $D_2$. This way, polynomials remain in factorized form during the
-     fraction simplifications. This yields smaller expressions when
-     reducing to the same denominator since common factors can be
-     canceled.
-
-  \item {\tt field\_simplify   [\term$_1$ {\ldots} \term$_n$]}
-     performs the simplification in the conclusion of the goal using
-    the equalities
-    defined by \term$_1$ {\ldots} \term$_n$.
-
-  \item {\tt field\_simplify   [\term$_1$ {\ldots} \term$_n$] $t_1$ \ldots
-$t_m$}
-     performs the simplification in the terms $t_1$ \ldots $t_m$
-    of the conclusion of the goal using
-    the equalities
-    defined by \term$_1$ {\ldots} \term$_n$. 
-
-  \item {\tt field\_simplify in $H$}  
-     performs the simplification in the assumption $H$.
-
-  \item {\tt field\_simplify   [\term$_1$ {\ldots} \term$_n$] in $H$}
-     performs the simplification in the assumption $H$ using
-    the equalities
-    defined by \term$_1$ {\ldots} \term$_n$. 
-
-  \item {\tt field\_simplify   [\term$_1$ {\ldots} \term$_n$] $t_1$ \ldots
-$t_m$ in $H$}
-     performs the simplification in the terms $t_1$ \ldots $t_n$
-    of the assumption $H$ using
-    the equalities
-    defined by \term$_1$ {\ldots} \term$_m$. 
-
-  \item {\tt field\_simplify\_eq}
-     performs the simplification in the conclusion of the goal removing
-     the denominator. $F_1 = F_2$
-     becomes $N_1 D_2 = N_2  D_1$.
-
-  \item {\tt field\_simplify\_eq   [\term$_1$ {\ldots} \term$_n$]}
-     performs the simplification in the conclusion of the goal using
-    the equalities
-    defined by \term$_1$ {\ldots} \term$_n$. 
-
-  \item {\tt field\_simplify\_eq} in $H$
-     performs the simplification in the assumption $H$.
-
-  \item {\tt field\_simplify\_eq   [\term$_1$ {\ldots} \term$_n$] in $H$}
-     performs the simplification in the assumption $H$ using
-    the equalities
-    defined by \term$_1$ {\ldots} \term$_n$. 
-\end{Variants}
-
-\asection{Adding a new field structure
-\comindex{Add Field}}
+The tactic must be loaded by ``Require Import Field``. New field
+structures can be declared to the system with the ``Add Field`` command
+(see below). The field of real numbers is defined in module ``RealField``
+(in ``plugins/setoid_ring``). It is exported by module ``Rbase``, so
+that requiring ``Rbase`` or ``Reals`` is enough to use the field tactics on
+real numbers. Rational numbers in canonical form are also declared as
+a field in module ``Qcanon``.
+
+
+.. example::
+
+  .. coqtop:: all
+
+    Require Import Reals.
+    Open Scope R_scope.
+    Goal forall x,
+           x <> 0 -> (1 - 1 / x) * x - x + 1 = 0.
+    intros; field; auto.
+    Goal forall x y, 
+           y <> 0 -> y = x -> x / y = 1.
+    intros x y H H1; field [H1]; auto.
+
+
+Variants:
+
+
+field [|term_1| `...` |term_n| ] 
```

Comment:
Is this sufficient to get an entry in the tactic index? Otherwise I'm worried that the tactic index is lost. The same remark applies to other tactics in this chapter.